### PR TITLE
yml and desc cleanup

### PR DESCRIPTION
**Pull request by Arsenikr**
cleaned unneccessary line feeds from yml
modified the description

## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Integrations/Ipstack/Ipstack_description.md [hunk 1]
```diff
@@ -1,98 +1,6 @@
...
-## Known Limitations
----
-
-## Troubleshooting
----
-
-
-## Possible Errors (DO NOT PUBLISH ON ZENDESK):
+**api_key**  
+api key to authenticate against ipstack
```
Comment (anara123) [inline Integrations/Ipstack/Ipstack_description.md]: where/how do I get this api key?

Comment (Arsenikr) [reply Integrations/Ipstack/Ipstack_description.md]: will elaborate. you get it from the UI

**Comment by anara123** [comment]
@kirbles19 please review https://github.com/demisto/content/blob/ipstack_integration/Integrations/Ipstack/Ipstack_description.md

**Comment by kirbles19** [comment]
@Arsenikr All done reviewing the .yml and .md files.

**Pull request merged**